packages/vx-gradient/src/gradients/GradientOrangeRed.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import LinearGradient, { LinearGradientProps } from './LinearGradient';
+
+/**
+ * All props pass through to `<LinearGradient {...props} />`
+ */
+export default function GradientOrangeRed({
```

Comment:
I am curious if we need one class per color scheme?

`GradientOrangeRed`, `GradientPinkBlue`, ...

Could we do 

```
<LinearGradient scheme="OrangeRed" />
```

or this if prefer to separate the class?

```
<PresetLinearGradient preset="OrangeRed" />
```

Reply:
I agree that we don't need a class per scheme, I like the `<LinearGradient scheme="OrangeRed" />` proposal.

That said, I'd like to avoid major re-factors or breaking changes along with the TypeScript refactor so would like to punt on that for now.